### PR TITLE
chore: Added example of scheduled events with lambda functions

### PR DESCRIPTION
**Pull request by lucaslacerdacl**
## How Has This Been Tested?
- [ ] I have tested and validated these changes using one or more of the provided `examples/*` projects

### CODE REVIEW

File: examples/with-lambda-scheduling/main.tf [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Actual Eventbridge
+####################
+module "eventbridge" {
+  source = "../../"
+
+  # Schedules can only be created on default bus
+  create_bus = false
+
+  # Fire every five minutes
+  rules = {
```
Comment (antonbabenko) [inline examples/with-lambda-scheduling/main.tf:11]: Let's merge these `rules`, `targets` with the rest of the things inside of the `complete` example. I think there is no need to have this as a separate example.

PS: Try to run an example yourself as a confirmation that it works as you expect.

File: README.md [hunk 2]
```diff
@@ -182,6 +182,34 @@ module "eventbridge_with_permissions" {
...
+
+```hcl
+module "eventbridge" {
+  source = "terraform-aws-modules/eventbridge/aws"
+
+  create_bus = false
+
+  rules = {
+    orders = {
+      description = "Trigger for a Lambda"
```
Comment (antonbabenko) [inline README.md:195]: Please fix formatting in the code snippet, too.

**Review by antonbabenko — CHANGES_REQUESTED** [review]
Looks good, but the main comment is that let's merge this example with the `complete` one.

File: examples/with-lambda-scheduling/main.tf (hunk 1 above)
Comment (lucaslacerdacl) [reply examples/with-lambda-scheduling/main.tf]: I added in the complete example as requested, but I also left a specific example for lambdas following the ECS pattern. Wouldn't it be interesting to keep a separate example for lambdas as in the other scenarios?

PS.: I ran the specific example of lambdas on my local machine, I needed to create a folder with a very basic code to define the code that would go to the lambda and also changed the variables of the modules that were being retrieved. Outra coisa importante é que para executar esse exemplo é necessário o Python 3.

File: README.md (hunk 2 above)
Comment (lucaslacerdacl) [reply README.md:195]: I changed the names according to what is in the full example and I didn't quite understand what should be fixed in lint. I followed the ECS example documentation that was already in the README.md file

**Comment by antonbabenko** [comment]
This PR is included in version 1.13.1 :tada:

**Pull request merged**